vcd/resource_vcd_vapp_firewall_rules.go

```diff
@@ -361,30 +363,33 @@ var errHelpVappNetworkFirewallRulesImport = fmt.Errorf(`resource id must be spec
 // Example list path (_the_id_string_): list@org-name.vdc-name.vapp-name
 // Note: the separator can be changed using Provider.import_separator or variable VCD_IMPORT_SEPARATOR
 func vappFirewallRuleImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	return vappNetworkRuleImport(d, meta, "vcd_vapp_firewall_rules")
+}
+func vappNetworkRuleImport(d *schema.ResourceData, meta interface{}, resourceType string) ([]*schema.ResourceData, error) {
 	var commandOrgName, orgName, vdcName, vappName string
 	resourceURI := strings.Split(d.Id(), ImportSeparator)
 
-	log.Printf("[DEBUG] importing vcd_vapp_firewall_rules resource with provided id %s", d.Id())
+	log.Printf("[DEBUG] importing %s resource with provided id %s", resourceType, d.Id())
 
 	if len(resourceURI) != 4 && len(resourceURI) != 3 {
-		return nil, errHelpVappNetworkFirewallRulesImport
+		return nil, errHelpVappNetworkRulesImport
 	}
 	if strings.Contains(d.Id(), "list@") {
 		commandOrgName, vdcName, vappName = resourceURI[0], resourceURI[1], resourceURI[2]
 		commandOrgNameSplit := strings.Split(commandOrgName, "@")
 		if len(commandOrgNameSplit) != 2 {
-			return nil, errHelpVappNetworkFirewallRulesImport
+			return nil, errHelpVappNetworkRulesImport
 		}
 		orgName = commandOrgNameSplit[1]
 		return listVappNetworksForImport(meta, orgName, vdcName, vappName)
 	} else {
 		orgName, vdcName, vappId, networkId := resourceURI[0], resourceURI[1], resourceURI[2], resourceURI[3]
-		return getFirewallRules(d, meta, orgName, vdcName, vappId, networkId)
+		return getRules(d, meta, orgName, vdcName, vappId, networkId)
 	}
 
 }
 
-func getFirewallRules(d *schema.ResourceData, meta interface{}, orgName, vdcName, vappId, networkId string) ([]*schema.ResourceData, error) {
+func getRules(d *schema.ResourceData, meta interface{}, orgName, vdcName, vappId, networkId string) ([]*schema.ResourceData, error) {
```

Comment:
Maybe more consistent:
```suggestion
func getNetworkRules(d *schema.ResourceData, meta interface{}, orgName, vdcName, vappId, networkId string) ([]*schema.ResourceData, error) {
```

Reply:
changed